src/configure/configure.ts

```diff
@@ -118,37 +119,21 @@ class Orchestrator {
 
     private async getAzureResource(targetType: TargetResourceType) {
         var azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(targetType);
-        var resource = null;
-        try {
-            resource = await azureResourceSelector.getAzureResource(this.inputs);
-            return resource;
-        }
-        catch (err) {
-            if (err.message === Messages.setupAlreadyConfigured) {
-                this.continueOrchestration = false;
-                return;
-            }
-            else {
-                throw err;
-            }
-        }
+        this.inputs.targetResource.resource = await azureResourceSelector.getAzureResource(this.inputs);
     }
 
-    private async selectTemplate(resource: GenericResource) {
+    private async selectTemplate(resource: GenericResource): Promise<void> {
         switch (resource.type) {
             case TargetResourceType.AKS:
-                let templateParameterHelper = new TemplateParameterHelper();
-                let template = this.inputs.potentialTemplates[0] as LocalPipelineTemplate;
-                this.inputs.pipelineConfiguration.template = this.inputs.potentialTemplates[0];
-                await templateParameterHelper.setParameters(template.parameters, this.inputs);
+                this.inputs.pipelineConfiguration.template = this.inputs.potentialTemplates.find((template) => template.templateType === TemplateType.LOCAL);
                 break;
-
             case TargetResourceType.WebApp:
                 var shortlistedTemplates = [];
                 shortlistedTemplates = this.inputs.potentialTemplates.filter((template) => template.targetKind === resource.kind);
                 if (!!shortlistedTemplates && shortlistedTemplates.length > 1) {
                     this.inputs.pipelineConfiguration.template = shortlistedTemplates.find((template) => template.templateType === TemplateType.REMOTE);
-                } else if (!!shortlistedTemplates) {
+                }
+                else if (!!shortlistedTemplates) {
```

Comment:
why move else to new line? If I know correctly, else should be in same line as preceding closing brace for nodejs/javascript. No?